design/ceph/object-store.md

```diff
@@ -95,51 +92,16 @@ The RGW service can be configured to listen on both http and https by specifying
     instances: 1
 ```
 
-### Realms, Zone Groups, and Zones
+### Realm
+The realm settings correspond to the Ceph RGW Multisite configuration.
 
-By default, the object store will be created independently from any other object stores and replication to another object store will not be configured. This done by creating a new realm, zone group, and zone all with the name of the new object store. The zone group and zone are tagged as the `master`. If this is the first object store in the cluster, the realm, zone group, and zone will also be marked as the default.
```

Comment:
This will be still valid right?

Reply:
Agreed, I thought this behavior would continue so we would have backward compatibility of the object stores.